### PR TITLE
Form section

### PR DESCRIPTION
**Pull request by lojacho**
In this pull request, I was able to:

  -  Create a form in the index.html using the pair programming method.

  -   Stylized this section in  HTML with CSS using flexbox the pair programming method.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,6 +196,49 @@ <h1>About me</h1>
...
+					<button class="formButton footerTopButton" type="submit">Get in touch</button>
+				</div>
+			</form>
+			<div class="mediaForm">
+				<nav>
+					<ul class="media">
+						<li><a href="#"><img src="./icons/Gitf.png" alt="github"></a></li>
+						<li><a href="#"><img src="./icons/LinkedInf.png" alt="linkedIn"></a></li>
+						<li><a href="#"><img src="./icons/fingerf.png" alt="crossfingers"></a></li>
+						<li><a href="#"><img src="./icons/Twitterf.png" alt="twitter"></a></li>
```
Comment (Lizdev-05) [inline index.html:239]: Great job 👍 @dgcuenca , Kindly give your form page some padding to make your user interface meet the required [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
**Your design 👇**
padding

**Microverse design **
pad

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @dgcuenca, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights

- All linters checks are green ✔️
- Descriptive PR title and summary✔
- The repository is documented in a professional way 👍
- The form is submitted 💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
-styling your placeholder field

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lojacho** [comment]
In this pull request, I was able to:

-Give padding to the form.
-Style placeholders.

**Review by omarsalem7 — APPROVED** [review]
Hi @dgcuenca 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**